docs/en/samples/accelerate_different_storage.md

```diff
@@ -0,0 +1,111 @@
+# DEMO - Accelerate different Stowith Fluid
+
+We need to add additional configuration to Alluxio for accessing the storage normally, when using different **underlying storage services** as the underlying storage system of Alluxio.
+
+This document shows how to declaratively complete the special configuration required by Alluxio in Fluid to access different storage services such as S3, HDFS, Ceph S3, PV, and MinIo. Please visit [Amazon AWS S3 - Alluxio v2.8.1 (stable) Documentation](https://docs.alluxio.io/os/user/stable/en/ufs/S3.html) for more  information.
+
+## Prerequisites
+
+- Before everything we are going to do, please refer to [Installation Guide](../userguide/install.md) to install Fluid on your Kubernetes Cluster, and make sure all the components used by Fluid are ready like this:
+
+    ~~~ shell
+    $ kubectl get pod -n fluid-system
+    NAME                                  READY   STATUS    RESTARTS   AGE
+    alluxioruntime-controller-5b64fdbbb-84pc6   1/1     Running   0          8h
+    csi-nodeplugin-fluid-fwgjh                  2/2     Running   0          8h
+    csi-nodeplugin-fluid-ll8bq                  2/2     Running   0          8h
+    dataset-controller-5b7848dbbb-n44dj         1/1     Running   0          8h
+    ~~~
+
+- The Storage which can be visited.
+
+## Configuration
+
+**Create Dataset Resource Object**
+
+``` yaml
+$ cat << EOF > dataset.yaml
+apiVersion: data.fluid.io/v1alpha1
+kind: Dataset
+metadata:
+  name: my-hdfs
+spec:
+  mounts:
```

Comment:
Can you add more mountpoints? Is subpath supported?

Reply:
OK, I can add more mountpoints.